conda/gateways/repodata/jlap/fetch.py

```diff
@@ -249,7 +249,8 @@ def download_and_hash(
         if is_zst:
             decompressor = zstandard.ZstdDecompressor()
             writer = decompressor.stream_writer(
-                HashWriter(dest_path.open("wb"), hasher), closefd=True  # type: ignore
+                HashWriter(dest_path.open("wb"), hasher),
+                closefd=True,
```

Comment:
do we need that comment?

Reply:
Yes, IIRC the checker doesn't understand that HashWriter is something you should be able to pass to stream_writer.

Reply:
my IDE didn't complain when I removed it 🤷🏻‍♂️ 

@dholth?